code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -52,6 +52,8 @@
 	var/list/mob/living/carbon/human/sac_targets
 	/// List of all sacrifice target's names, used for end of round report
 	var/list/all_sac_targets = list()
+	/// List of WEAKREFs to our created blades
+	var/list/created_blades = list()
```

Comment:
I feel like we shouldn't be tracking blades here, we have a knowledge datum that is already handling that for us.

You should just be able to find `/datum/heretic_knowledge/limited_amount/starting` in the heretic's knowledge list and check *it's* `created_items` list.